nvflare/fuel/f3/cellnet/cell.py

```diff
@@ -321,15 +346,17 @@ def _send_one_request(
 
             # waiting for receiving first byte
             self.logger.debug(f"{req_id=}: entering remote process wait {timeout=}")
-            if not waiter.in_receiving.wait(timeout):
-                self.logger.debug(f"{req_id=}: remote processing timeout {timeout=}")
+
+            waiter_rc = conditional_wait(waiter.in_receiving, timeout, abort_signal)
+            if waiter_rc != WaiterRC.IS_SET:
+                self.logger.debug(f"{req_id=}: remote processing timeout {timeout=} {waiter_rc=}")
```

Comment:
Do we want different debug log for WaiterRC.ABORTED?

Reply:
I think this is enough since waiter_rc is logged already.